device_calendar/ios/Classes/SwiftDeviceCalendarPlugin.swift

```diff
@@ -159,6 +171,33 @@ public class SwiftDeviceCalendarPlugin: NSObject, FlutterPlugin {
             }
             
         }
+        var recurrenceRule: RecurrenceRule?
+        if(ekEvent.hasRecurrenceRules) {
+            let ekRecurrenceRule = ekEvent.recurrenceRules![0]
+            var frequency: Int64
+            switch ekRecurrenceRule.frequency {
+            case EKRecurrenceFrequency.daily:
```

Comment:
Formatting: shouldn't the `case` statements be indented?

Reply:
Swift formatting is done differently and so are the switch statements (note curly brace instead of colon). The auto-format will undo the indentation. If you see this doc you'll see it's consistent with what's in the Swift docs https://docs.swift.org/swift-book/LanguageGuide/ControlFlow.html

Reply:
👍